code/modules/admin/admin.dm

```diff
@@ -1328,7 +1328,7 @@ var/global/floorIsLava = 0
 	NM.ui_interact(usr)
 
 /client/proc/update_mob_sprite(mob/living/carbon/human/H as mob)
-	set category = "Admin"
+	set category = "Server"
```

Comment:
This is `Server` how?

Reply:
generally speaking, affects server (as far as I'm aware)

Reply:
That sounds like something for Debug tab

Reply:
It does not affect server, put it back.